### ISSUE TITLE
Rollup of a Rollup API Bug

### ISSUE THREAD

**Issue by yujanshrestha** [innolitics/n2y#41]
This is a low priority issue because it is better handled on the Notion template side. However, I wrote up a description to make you guys aware of it, and a potential way to fix it in n2y.

If it is easy to fix and does not significantly affect the performance of n2y then I say we fix it. Otherwise, if it requires us make an API call for every single page in the database, it probably is not worth it. Notion API devs may end up fixing it before we need this.

tldr: What you see in the Notion UI is not always what you get in the API.

See details and replication instruction here:

https://www.notion.so/innolitics/Rollup-on-Rollup-2eadf45fc5d74677a82cfa033017b7f5
